integration_tests/test_mesh_lens_correction.py

```diff
@@ -147,7 +267,9 @@ def test_make_mask(tmpdir_factory):
                 basename=None)
 
 
-def test_mesh_lens_correction(render, tmpdir_factory):
+def test_mesh_lens_correction(
+        render, tmpdir_factory, raw_lens_stack_2, raw_lens_matches_2):
+
     outdir = str(tmpdir_factory.mktemp("mesh_lens"))
     out_html_dir = outdir
     example_for_input = dict(example)
```

Comment:
I don't know how it was done previously in this test, but we do a deep copy in all our other tests. It would be confusion free on the input data if a deep copy is done here (and in subsequent tests).

Reply:
good point, these were quite old